adapters/appnexus/appnexus.go

```diff
@@ -462,7 +475,18 @@ func getMediaTypeForBid(bid *openrtb.Bid) (openrtb_ext.BidType, error) {
 	case 3:
 		return openrtb_ext.BidTypeNative, nil
 	default:
-		return "", fmt.Errorf("Unrecognized bid_ad_type in response from appnexus: %d", impExt.Appnexus.BidType)
+		return "", fmt.Errorf("Unrecognized bid_ad_type in response from appnexus: %d", bid.Appnexus.BidType)
+	}
+}
+
+// getIabCategoryForBid maps an appnexus brand id to an IAB category.
+func (a *AppNexusAdapter) getIabCategoryForBid(bid *appnexusBidExt) (string, error) {
```

Comment:
Is there a unit test for this?

Reply:
There aren't direct unit tests for this function, but it does get executed by the test cases in the appnexustest/exemplary folder. I will be making sure test cases there cover both cases where the category is found in the mapping and also cases where it's not found.